src/main/java/net/dv8tion/jda/internal/JDAImpl.java

```diff
@@ -906,6 +906,28 @@ public GuildActionImpl createGuild(@Nonnull String name)
         return new GuildActionImpl(this, name);
     }
 
+    @Nonnull
+    @Override
+    public RestAction<Void> createGuildFromTemplate(@Nonnull String code, @Nonnull String name, Icon icon)
+    {
+        if (guildCache.size() >= 10)
+            throw new IllegalStateException("Cannot create a Guild with a Bot in 10 or more guilds!");
+
+        Checks.notBlank(code, "Template code");
+        Checks.notBlank(name, "Name");
+        name = name.trim();
+        Checks.check(name.length() >= 2 && name.length() <= 100, "Name must have 2-100 characters in length!");
```

Comment:
Use notLonger

Reply:
sure, but what about the lower bound of 2 characters?

Reply:
Is it really 2 characters? If so then you can add a `notShorter` equivalent I suppose.

Reply:
I can't reproduce, I can set one as a user with 1 character long just fine. Why would it be different for bots 🤔

Reply:
the discord docs say it's 2-100 characters so not sure

Reply:
Other places are out of scope for this PR.

Reply:
resolved, the lower limit is one character for both name and description